src/main/java/uk/gov/hmcts/reform/prl/services/managedocuments/ManageDocumentsService.java

```diff
@@ -490,6 +492,9 @@ private Document getNewUploadedDocument(Document document,
 
     public Document getQuarantineDocumentForUploader(String uploadedBy,
                                                      QuarantineLegalDoc quarantineLegalDoc) {
+        if (null == uploadedBy) {
```

Comment:
Please roll it back as its not required for other envs

Reply:
rolled back as requested